src/components/InfoCard/InfoCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useCallback, useState } from 'react';
+import { UnsupportedChainIdError, useStarknetReact } from '@web3-starknet-react/core';
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { SvgIcon } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import Box from '@mui/material/Box';
+import { Link, NavLink, useLocation } from 'react-router-dom';
+
+import { InfoLabel, InfoBox } from './InfoCard.styles';
+import icon from '../../resources/images/infoicon.png';
+
+const InfoCard = () => {
+
+  return (
+    <InfoBox>
+      <Box style={{textAlign: 'end', paddingRight: '10px', paddingTop: '10px'}}>
+        <img src={icon} />
```

Comment:
please save it as svg and use SvgIcon component like here - https://github.com/jediswaplabs/personal-profile/blob/master/src/components/Header/Header.jsx#L75
Here's the doc - https://mui.com/material-ui/icons/#svgicon